.github/workflows/ci.yml

```diff
@@ -7,122 +50,197 @@ on:
   pull_request:
```

Comment:
Do we want to add support for merge queues here?
See also https://github.com/linebender/vello/pull/647 in that case.

Reply:
Updated for that.